grepos/build.gradle

```diff
@@ -6,6 +6,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.1.2'
+        classpath 'me.tatarka:gradle-retrolambda:3.2.5'
```

Comment:
This is just an additional info; currently, this gradle-retrolambda library itself uses old version of retrolambda by default, so if we were to reference to the latest retrolambda, we need to `configure` it. 
https://github.com/evant/gradle-retrolambda#using-a-different-version-of-the-retrolambdajar